src/f4enix/input/migjorn_proxies.py

```
@@ -0,0 +1,214 @@
+    _model : migjorn.Model
+        the underlying migjorn model whose transforms are proxied.
+    """
+
+    def __init__(self, model: migjorn.Model) -> None:
+        self._model = model
+
+    def __getitem__(self, key: str) -> migjorn.Transform:
+        try:
+            tid = int(key.upper().lstrip("*").removeprefix("TR"))
+        except ValueError:
+            raise KeyError(key)
```

Ruff (0.16.2)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)